### DISCUSSION TITLE
Question of modeling

### DISCUSSION THREAD

**Discussion by xjlcode** [CURENT/andes#446] [Data Format]
Dear Dr.Cui,
I am writing to inquire about the model libraries in andes, particularly regarding the PQ model. I have noticed that the PQ model used in Andes is not consistent with the well-known power flow calculation model. I am curious to know where the inspiration for this model came from and what references or textbooks were used in its development.

Best regards，
xjl

**Comment by cuihantao** [comment]
Once again, we could better help you if your question is more specific. Which part of the equations is not consistent with the well-known models, and what are in the well-known models?
